src/odemis/driver/xt_client.py

```diff
@@ -174,6 +174,18 @@ def set_scan_mode(self, mode):
             self.server._pyroClaimOwnership()
             self.server.set_scan_mode(mode)
 
+    def get_scan_mode(self):
+        """
+        Get the scan mode.
+        Returns
+        -------
+        mode: str
+            Name of desired scan mode, one of: unknown, external, full_frame, spot, or line.
```

Comment:
desired -> set? As you don't want to set a "desired" mode, but want to know which one is set.